Rlecellier/cunningham (#1964)
openfun/richie: UPGRADE.md

## Unreleased

- palette.scss have been removed. Legacy colors are now defined in cunningham.cjs. 

>   `yarn build-theme` cmd generate variables files scss/cunningham-tokens.scss and /utils/cunningham-tokens.ts.

Reviewer: `command generates tokens files`

## Unreleased

- Legacy colors are now defined in cunningham.cjs instead of palette.scss.
  `yarn build-theme` command generates tokens files scss/cunningham-tokens.scss and /utils/cunningham-tokens.ts.

>   $palette variale is now deprecated and will be removed when cookiecutter website handle $colors.

Reviewer 2: ```suggestion
  $palette variable is now deprecated and will be removed in the next major release.
```
